offline/QA/Jet/RhosinEvent.cc

```diff
@@ -138,23 +140,61 @@ int RhosinEvent::process_event(PHCompositeNode* topNode)
     TowerRho* towerrho = findNode::getClass<TowerRhov1>(topNode, m_area_rho_node);
     if (!towerrho)
     {
-      std::cout << "RhosinEvent::process_event - Error can not find towerrho " << m_area_rho_node << std::endl;
+      std::cout << "RhosinEvent::process_event - Warning can not find towerrho " << m_area_rho_node << std::endl;
+      //exit(-1);
+    }
+    else{
+      h1_area_rho->Fill(towerrho->get_rho());
+      h1_area_rho_sigma->Fill(towerrho->get_sigma());
+    }
+
+    EventRho* eventrho = findNode::getClass<EventRhov1>(topNode, m_area_rho_node);
+    if (!eventrho)
+    {
+      std::cout << "RhosinEvent::process_event - Warning can not find eventrho " << m_area_rho_node << std::endl;
+      //exit(-1);
+    }
+    else{
+      h1_area_rho->Fill(eventrho->get_rho());
+      h1_area_rho_sigma->Fill(eventrho->get_sigma());
+      std::cout << "RhosinEvent::process_event - Found eventrho " << m_area_rho_node <<". Rho = "<<eventrho->get_rho()<<"  rho_sigma "<<eventrho->get_sigma()<< std::endl;
+    }
+    if (!eventrho && !towerrho)
+    {
+      std::cout << "RhosinEvent::process_event - Error can not find neither towerrho nor eventrho for" << m_area_rho_node << std::endl;
       exit(-1);
```

Comment:
In line with above, I would suggest returning here...
```suggestion
      return Fun4AllReturnCodes::EVENT_OK;
```